lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1230,17 +1357,19 @@ struct FpToFpOpConversion
         // F8 -> F16
         {{F8E4M3B15TyID, F16TyID}, Fp8E4M3B15_to_Fp16},
         {{F8E4M3FNTyID, F16TyID}, Fp8E4M3B15x4_to_Fp16},
-        {{F8E4M3TyID, F16TyID}, Fp8E4M3_to_Fp16},
+        {{F8E4M3FNUZTyID, F16TyID}, Fp8E4M3FNUZ_to_Fp16},
         {{F8E5M2TyID, F16TyID}, Fp8E5M2_to_Fp16},
+        {{F8E5M2FNUZTyID, F16TyID}, Fp8E5M2FNUZ_to_Fp16},
         // F16 -> F8
 #ifdef USE_ROCM
         {{F16TyID, F8E4M3B15TyID}, Fp16_to_Fp8E4M3B15},
 #else
         {{F16TyID, F8E4M3B15TyID}, Fp16_to_Fp8E4M3B15(computeCapability >= 80)},
 #endif
         {{F16TyID, F8E4M3FNTyID}, Fp16_to_Fp8E4M3B15x4},
```

Comment:
I think this line should be:
{{F16TyID, F8E4M3FNTyID}, Fp16_to_Fp8E4M3},

Reply:
Can we change this? This type is used by nvidia and they defined the type format.

Reply:
Yeah, it is using the nvidia type but there is a mismatch between the triton and mlir. Phil said that he was going to clean it up but he was on vacation. So I was under the impression that we would clean it up experimentally and might try to upstream it. But if we don't have a plan to upstream that, that's fine and we might just wait for the update from Phil.

Reply:
Since we do not actually use these data types and we do not know how upstream is going to handle these types, so we can just wait for upstream to do that.